### ISSUE TITLE
Canvas Angular integration: CanvasHost, bridge service, panels

### ISSUE THREAD

**Issue by zenvanexus** [nebucloud/zeld-ui#9]
## Summary

Bridge the pure TypeScript canvas engine into Angular — host component, signals bridge, and UI panels.

## Location

`src/app/canvas/`

### CanvasHostComponent (`canvas-host.component.ts`)
- Hosts the `<canvas>` element
- `ngAfterViewInit`: creates `CanvasEngine` instance via `viewChild`, passes to `CanvasBridgeService`
- `ngOnDestroy`: calls `engine.unmount()`, `bridge.detach()`
- `OnPush` change detection (canvas self-renders via rAF)
- Handles canvas resize on container resize (ResizeObserver)

### CanvasBridgeService (`services/canvas-bridge.service.ts`)
- `attach(engine: CanvasEngine)` / `detach()`
- Subscribes to `EngineEvent`, runs them inside `NgZone.run()` to trigger Angular change detection
- Translates engine events into `ComponentGraphService` signal updates
- Translates `ComponentGraphService` commands into `engine.dispatch()`

### ComponentGraphService (`services/component-graph.service.ts`)
- `graph: WritableSignal<SceneGraph>` — shared source of truth
- `selected: computed()` — currently selected nodes
- `hoveredNode: WritableSignal<string | null>`
- `addNode()`, `updateNode()`, `deleteNode()`, `reorderNode()` — dispatch to engine and update signals
- Undo/redo stack (command pattern)

### UI Panels
- **PropertyPanelComponent** — reads `selected`, displays/edits node properties (position, size, style, component props)
- **LayersPanelComponent** — tree view of scene graph, reorder via drag, visibility toggles
- **ComponentPaletteComponent** — list of available components from design system kits, drag to add to canvas
- **AlignmentToolbarComponent** — align/distribute selected nodes

### ProjectCanvasComponent (`pages/project-canvas/`)
- Layout: canvas center + panels (left: layers, right: properties, top: toolbar, bottom: component palette)
- Composes all canvas-related components
- Chat sidebar for AI generation within the canvas view

## Architecture
```
Angular Zone                      Engine (pure TS)
──────────────                    ────────────────
PropertyPanel ──▶ ComponentGraphService ──▶ CanvasBridge ──▶ engine.dispatch()
LayersPanel   ◀── ComponentGraphService ◀── CanvasBridge ◀── engine.on(event)
```

## Dependencies

- Depends on #1 (scaffold), #5-#8 (engine), #2 (core services)
